analysis/jira__daily_issue_status_category_analysis.sql

```diff
@@ -0,0 +1,66 @@
+-- grab non hard-coded issue_field_history columns
+{% set issue_field_history_columns = var('issue_field_history_columns', []) %}
+
+
+with field_history as (
+ select *
+ from {{ ref('jira__timestamp_issue_field_history') }}
+),
+
+status_transitions as (
+ select *
+ from {{ ref('jira__issue_status_transitions') }}
+),
+
+---if you prefer to keep anayltics at the status level,substitute 'status' for 'status_category' throughout this model
```

Comment:
```suggestion
---if you prefer to keep analytics at the status level, substitute 'status' for 'status_category' throughout this model
```

Reply:
We could also consider including both status and status_category_name. As long as they have the compiled code, they can remove whatever column they want.